fleetctl/fleetctl.go

```diff
@@ -482,6 +482,36 @@ func getChecker() *ssh.HostKeyChecker {
 	return ssh.NewHostKeyChecker(keyFile)
 }
 
+func getUnitFile(file string) (*unit.UnitFile, error) {
+	var uf *unit.UnitFile
+	name := unitNameMangle(file)
+
+	log.Debugf("Looking up for Unit(%s) or its corresponding template", name)
```

Comment:
I assume they will show up when debug is set?! just followed up the current behaviour.
I would say yeh we may need them especially with all the concurrency problems, having these would help. Now if they show up by default then yeh that's another problem, that perhaps we should fix in another place ?